src/Repository/SaferPayCardAliasRepository.php

```diff
@@ -74,4 +74,14 @@ public function getSavedCardsByCustomerId($customerId)
 
         return Db::getInstance()->executeS($query);
     }
+
+    public function getSavedCardsNumbersByCustomerId($customerId)
+    {
+        $query = new DbQuery();
+        $query->select('`card_number`');
+        $query->from('saferpay_card_alias');
+        $query->where('id_customer = "' . (int) $customerId . '"');
+
+        return Db::getInstance()->executeS($query);
```

Comment:
What if result is null ? Make sure have consistent return types, would be nice to have phpDocs above methods

You should check is result legit, if no return empty array